keras_cv/models/object_detection/yolox/__internal__/binary_crossentropy.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2022 The KerasCV Authors
```

Comment:
Suggestion: remove all yolox/__internal__ at this PR, so it just focus on giou and iou, for faster review and merge

Reply:
Meanwhile what is the BCE loss trying to fix, compared to keras.BCE loss?

Reply:
For this one `axis=None` means no mean over any axis.

Reply:
Alright, I will remove `binary_crossentropy` from this one.

Reply:
I think we should have a special string to ignore axis.  axis=None usually means mean over all axes in TF

Reply:
We had a thread at https://github.com/keras-team/keras/issues/17207